funnel/templates/notifications/layout_email.html.jinja2

```diff
@@ -1,45 +1,511 @@
 <!DOCTYPE html>
-<html lang="{{ get_locale() }}">
-  <head>
+<html lang="{{ get_locale() }}" xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+<head>
+    <meta charset="utf-8"> {# utf-8 works for most cases #}
+    <meta name="viewport" content="width=device-width"> {# Forcing initial-scale shouldn't be necessary #}
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"> {# Use the latest (edge) version of IE rendering engine #}
+    <meta name="x-apple-disable-message-reformatting">  {# Disable auto-scale in iOS 10 Mail entirely #}
+    <meta name="format-detection" content="telephone=no,address=no,email=no,date=no,url=no"> {# Tell iOS not to automatically link certain text strings. #}
```

Comment:
This may be a useful thing to leave in the default state.

Reply:
Ok. Should I remove all of them? First three look like they could be useful.